### ISSUE TITLE
Images in the 'Other Sports' container lack alt text

### ISSUE THREAD

**Issue by bryophyta** [guardian/frontend#25245]
There is an onwards container with the title 'Other Sports' on pages like the [uk/sport](https://www.theguardian.com/uk/sport) where the images lack alt text (i.e. not even an empty `alt` attribute).

**Comment by jlieb10** [comment]
To do: figure out with the tools team and sports editorial how the "other sports" container is built/generated so we can add the alt text

**Comment by bryophyta** [comment]
I think I've fixed this (at least somewhat) in [this PR](https://github.com/guardian/frontend/pull/25372) by adding an empty `alt` tag to the image element that's used. It needs to be tested, though, which requires adding a specific container to CODE fronts. We're not currently clear on how to do this as a team so I've reached out to Central Production. Blocked until we get a response from them (or if anyone else knows how to do it!)

**Comment by bryophyta** [comment]
(The fix mentioned above has been tested and merged now.)
